cmake/define-test.cmake

```diff
@@ -187,12 +188,18 @@ function(define_test)
         ${ORIGINAL_INCLUDE_DIR} ${REWRITTEN_INCLUDE_DIR}
         OUTPUT_VARIABLE COPIED_HEADERS)
     add_custom_command(
-        OUTPUT ${COPIED_SRCS} ${COPIED_HEADERS}
+        OUTPUT ${COPIED_SRCS}
         COMMAND cp ${ORIGINAL_SRCS} ${CMAKE_CURRENT_BINARY_DIR}/
-        COMMAND mkdir -p ${REWRITTEN_INCLUDE_DIR}
-        COMMAND cp ${ORIGINAL_HEADERS} ${REWRITTEN_INCLUDE_DIR}/
-        DEPENDS ${ORIGINAL_SRCS} ${ORIGINAL_HEADERS}
+        DEPENDS ${ORIGINAL_SRCS}
         WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+    if (NOT ORIGINAL_HEADERS STREQUAL "")
```

Comment:
Aren't you setting `ORIGINAL_TARGETS` instead of `ORIGINAL_HEADERS` in the permissive mode test?

Reply:
When calling `define_ia2_wrapper()`, yeah? It's just that this test doesn't have any headers and only a single compartment.